roles/tas_single_node/tasks/podman.yml

```diff
@@ -44,33 +45,38 @@
     - "{{ tas_single_node_tuf_image }}"
     - "{{ tas_single_node_netcat_image }}"
     - "{{ tas_single_node_nginx_image }}"
+    - "{{ tas_single_node_tsa_image }}"
   loop_control:
     index_var: idx
   when: enabled_vals[idx] | bool
 
-- name: Configure/Deploy nginx
-  ansible.builtin.include_tasks: podman/nginx.yml
+# - name: Configure/Deploy nginx
+#   ansible.builtin.include_tasks: podman/nginx.yml
```

Comment:
I think you commented out a lot of lines in this file for testing but forgot to uncomment?